packages/eslint-plugin/tests/rules/prefer-nullish-coalescing.test.ts

```diff
@@ -222,6 +222,18 @@ x || y;
       `,
       options: [{ ignorePrimitives: true }],
     })),
+    `
+      declare const x: any;
+      x || y;
+    `,
+    `
+      declare const x: unknown;
+      x || y;
+    `,
+    `
+      declare const x: never;
+      x || y;
+    `,
```

Comment:
[Testing] `y` is an error type (so, kind of `any`) in these tests. Even if that happens to not violate the rule now it's still not a great practice. They should be `declare`d too.

Reply:
Done, thx! https://github.com/typescript-eslint/typescript-eslint/pull/8262/commits/ce3d8e0c3d4d4850565cdf04dde85b71b09d3d58